lib/navigation-bar/index.tsx

```diff
@@ -126,13 +126,12 @@ export class NavigationBar extends Component<Props> {
 
 const mapStateToProps: S.MapState<StateProps> = ({
   settings,
-  ui: { dialogs, openedTag, showNavigation, showTrash },
+  ui: { dialogs, showNavigation, collection },
```

Comment:
Just a weird nit, we tend to (try to) put these in alphabetical order

Reply:
Will do, fixing :)

Reply:
@codebykat Alphabetized!